canonical-web-and-design/multipass.run
# Add Discourse module

## Done

Added discourse docs module, along with documentation layout scss and markup

## QA
- Check out this branch
- `./run` the project
- Visit http://0.0.0.0:8026/docs
- See that the content there is being pulled from https://discourse.ubuntu.com/t/multipass-documentation-outline/8294

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2146 & https://github.com/canonical-web-and-design/web-squad/issues/2147
